### DISCUSSION TITLE
Turning off stations

### DISCUSSION THREAD

**Discussion by renatogfilhinho** [ltsp/ltsp#871] [General]
Good morning! Please, I'm having difficulty turning off stations, when I click off, the station restarts. If anyone can help, I would be very grateful.

**Comment by alkisg** [comment]
Upload a video of the whole shutdown process.
It sounds like a kernel/ACPI issue, unrelated to LTSP.

Also if you can, try to install your distribution locally and see if it can shut down when LTSP isn't used. If it can't shutdown then either, then you should contact your distribution support channels.

**Reply by renatogfilhinho** [reply]
You're right, I tested with the same user on another station, LTSP has nothing to do with it. I apologize for taking up the space with the subject. Thanks.
